glustercli/cmd/utils.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 var (
-	validSizeFormat = regexp.MustCompile(`^([0-9]+)([GMKT])?$`)
+	validSizeFormat = regexp.MustCompile(`^([0-9]+)\s*([kKmMgGtT]?[iI]?[bB]?)$`)
```

Comment:
What're iI & bB for?

Reply:
to support units like `MB` or `MiB` , `kb`, `kib`